src/mappings/handlers/blockHandlers.ts

```diff
@@ -44,13 +45,15 @@ async function _handleBlock(block: SubstrateBlock): Promise<void> {
         await tranche.save()
       }
 
-      // Update loans outstanding debt
       const activeLoanData = await pool.getActiveLoanData()
       for (const loanId in activeLoanData) {
         const loan = await LoanService.getById(pool.pool.id, loanId)
         const { normalizedDebt, interestRate } = activeLoanData[loanId]
         await loan.updateOutstandingDebt(normalizedDebt, interestRate)
         await loan.save()
+
+        if (loan.loan.maturityDate > block.timestamp)
```

Comment:
I think this needs to be inverted? Right now it's checking if the maturity date is larger than the current timestamp, meaning the maturity date is in the future; while loans are overdue if the maturity date is in the past.
```suggestion
        if (loan.loan.maturityDate > block.timestamp)
```

Reply:
Definitely....